{{cookiecutter.github_repository}}/mkdocs.yml

```diff
@@ -14,6 +14,7 @@ nav:
 - Technical Specs:
   - Server Config: backend/server_config.md
   - Coding Rules: backend/coding_rules.md
+  - Database Schema: database-schema.svg
```

Comment:
How about putting this in backend folder?

Reply:
done